staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -67,13 +67,13 @@ func HTTPWrappersForConfig(config *Config, rt http.RoundTripper) (http.RoundTrip
 // DebugWrappers wraps a round tripper and logs based on the current log level.
 func DebugWrappers(rt http.RoundTripper) http.RoundTripper {
 	switch {
-	case bool(klog.V(9)):
+	case bool(klog.V(5)):
 		rt = newDebuggingRoundTripper(rt, debugCurlCommand, debugURLTiming, debugResponseHeaders)
-	case bool(klog.V(8)):
+	case bool(klog.V(4)):
 		rt = newDebuggingRoundTripper(rt, debugJustURL, debugRequestHeaders, debugResponseStatus, debugResponseHeaders)
-	case bool(klog.V(7)):
+	case bool(klog.V(3)):
 		rt = newDebuggingRoundTripper(rt, debugJustURL, debugRequestHeaders, debugResponseStatus)
-	case bool(klog.V(6)):
+	case bool(klog.V(2)):
 		rt = newDebuggingRoundTripper(rt, debugURLTiming)
 	}
```

Comment:
/hold

on this change... trace-level request logging is intentionally at much higher levels. mixing request-level trace logs with logic-level trace logs ("did x", "did y", etc) would make the logic-level trace logs almost impossible to consume

Reply:
For a switch on the log level such as this, is it then better to flatten it out to only be 4/5 verbosity (removing some of the cases), or just keep it outside of the typical 1-5 verbosity levels like it already is?

Reply:
Each level is useful for getting increasingly more trace information (url, headers, truncated body, full body). I don't see the benefit in flattening or removing cases.